lambda_code/utilities/create_operation_outcome.py

```diff
@@ -0,0 +1,27 @@
+import uuid
+
+def create_response(message, code):
```

Comment:
Doesn't appear to be used, hard codes "error" though that's not clear from the name, and otherwise duplicates `create_operation_outcome`. Mistakenly committed maybe?